tests/cmdline/commands/test_status.py

```diff
@@ -8,52 +8,28 @@
 # For further information please visit http://www.aiida.net               #
 ###########################################################################
 """Tests for `verdi status`."""
-from click.testing import CliRunner
-
-from aiida.backends.testbase import AiidaTestCase
 from aiida.cmdline.commands import cmd_status
+from aiida.cmdline.utils.echo import ExitCode
 
-from tests.utils.configuration import with_temporary_config_instance
-
-
-class TestVerdiStatus(AiidaTestCase):
-    """Tests for `verdi status`.
 
-    Note: The exit status may differ depending on the environment in which the tests are run.
-        Also cannot check for the exit status to see if connecting to all services worked, because
-        the profile might not be properly setup in this temporary config instance unittest.
-    """
+def test_status(run_cli_command):
```

Comment:
Nice! I was thinking of doing this, but wasn't sure of the right fixture: didn't notice this one